bin/asl-node-lookup

```diff
@@ -0,0 +1,266 @@
+#!/bin/bash
+
+ASL_NODE=""
+USE_NS=""
+VERBOSE=0
+
+# ---------- ---------- ---------- ---------- ----------
+
+if [[ ! -x /usr/bin/host ]]; then
+    echo "This command executes the \"host\" command to collect DNS information"
+    echo "about ASL nodes.  The \"host\" command is not available on this system."
+    if [[ -x /usr/bin/apt ]]; then
+	echo ""
+	echo "To install the \"host\" command, use \"apt install bind9-dnsutils\"."
+    fi
+    exit 1
+fi
```

Comment:
Is all of this necessary, since you are already marking the bind9 package as a dependency of ASL3 in the debian control file?

Reply:
I added this block to help those who might install the script without using the .deb package.